Sources/SpeziViews/Model/ViewState.swift

```diff
@@ -10,6 +10,52 @@ import Foundation
 
 
 /// Allows SwiftUI views to keep track of their state and communicate to outside views.
+///
+/// A `ViewState` provides a built-in mechanism for tracking the state of a Spezi UI component.
+/// A view can be in either an `idle`, `processing`, or `error` `ViewState`.
```

Comment:
I would suggest to note that a `ViewState` takes a `LocalizedError` as an associated value and would point to the Apple Documentation on this.

We should also add a small tip to our "best effort" matching of non-Localized errors in `AnyLocalizedError` in this package.